### DISCUSSION TITLE
openid.13a.com 完全进不去咋办

### DISCUSSION THREAD

**Discussion by Niceqingyao** [natfrp/wiki#171] [使用问题] [answered]
您正在使用 Nyatwork OpenID 备用登录接口。如果您可以正常访问 openid.13a.com，请不要使用此接口以免产生各种问题

**Comment by fakeboboliu** [comment, marked as answer]
请问访问 https://openid.13a.com 时具体的报错是什么呢？

如果您认为目前的描述已经是您能描述的极限了，请多贴几张截图。

**Reply by Niceqingyao** [reply]
就是网络连接错误 但是我本地网速没有任何问题的

**Reply by fakeboboliu** [reply]
> 如果您认为目前的描述已经是您能描述的极限了，请多贴几张截图。

**Reply by Niceqingyao** [reply]
![image](https://user-images.githubusercontent.com/49465672/189836209-abc66d76-971c-42b3-93fd-1ae2b97d7c12.png)

**Reply by fakeboboliu** [reply]
此问题通常为系统配置问题，请检查 DNS 和 Hosts。
